.gitignore

```diff
@@ -15,4 +15,4 @@ frontend/localhost.crt
 frontend/localhost.csr
 frontend/localhost.key
 frontend/localhost.pfx
-frontend/appsettings.Development.local.json
\ No newline at end of file
+frontend/appsettings.Local.json
```

Comment:
Dit gebruiken we niet dus hele lijn mag weg, tenzij we hem binnenkort wel gaan gebruiken